### PR TITLE
[failover] Demo of MySQL failover with HAProxy

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: failover/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  * Apache ZooKeeper
+  * Apache Kafka broker
+  * Apache Kafka Connect with Debezium MySQL Connector - the connector will connect to HAProxy
+
+## Demonstration
+
+Start the components and register Debezium to stream changes from the database
+```
+export DEBEZIUM_VERSION=0.9
+docker-compose -f docker-compose-mysql.yaml up --build
```
Comment (gunnarmorling) [inline failover/README.md]: I'd suggest to rename the file to _docker-compose.yaml_, allowing to omit the `-f` option.

File: failover/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  INSERT INTO customers VALUES (default, 'John','Doe','john.doe@example.com');
+  INSERT INTO customers VALUES (default, 'Jane','Doe','jane.doe@example.com');
+
+# Check UUID in change message, the 'source' will contain field "gtid":"50303655-f22a-11e8-92a5-0242ac1d0003:2"
+docker-compose -f docker-compose-mysql.yaml exec kafka /kafka/bin/kafka-console-consumer.sh --bootstrap-server kafka:9092 --from-beginning --property print.key=true --topic dbserver1.inventory.customers
+```
+
+Stop the primary server
+```
+docker-compose -f docker-compose-mysql.yaml stop
```
Comment (gunnarmorling) [inline failover/README.md]: This stops everything, no?

File: failover/README.md (hunk 1 above)
Comment (jpechane) [reply failover/README.md]: @gunnarmorling Right, changed

File: failover/README.md (hunk 2 above)
Comment (jpechane) [reply failover/README.md]: Thanks for spotting!

**Comment by jpechane** [comment]
@gunnarmorling Comments incroporated, regarding the blogpost, yes I intend to write one

**Comment by gunnarmorling** [comment]
And merged. Thanks, @jpechane! Can't wait to read that blog post, I think it's a super-interesting topic for many users.

**Pull request merged**